extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ParquetFileWriter.java

```diff
@@ -45,7 +44,7 @@ public final class ParquetFileWriter implements AutoCloseable {
 
     public ParquetFileWriter(
             final URI dest,
-            final OutputStream destOutputStream,
+            final CountingOutputStream countingOutput,
```

Comment:
If we do this, we need to elevate guava dependency from `implementation` to `api`. The (maybe preferable?) alternative to to add `ParquetFileWriter#getCount` or something similar.